### ISSUE TITLE
Uhhhhhhh

### ISSUE THREAD

**Issue by victti** [Gr8Jerryto/Gr8Pepi#1]
Since Github doesn't have a way to PM nor you had emails nor were on the discord server, I'll just ask it here. Do I know you? I remember playing with Pinion on Aerilon, is it really you? (hu3?)

**Comment by Gr8Jerryto** [comment]
Sim, nós éramos os únicos BRs dentro da sector control, jogávamos bastante,
grindamos nosso caminho pra cima, tinha o DarkLee, que era russo que viva
no usa, e era super competitivo comigo, nosso líder era Devil_Redemption e
Devil_Redemption 2, um dos meus maiores feitos foi grindar agua com boost
por 24H naquela Fenrir a qual você perguntou se podia gravar o video do
pvp. Uma vez você trocou de conta com alguém por uma Surtur (que nunca tive
a honra de ter). Nossa ala era pequena e já tava numa época estranha pro
Bsgo. Havia um cara não me lembro o nome que disse que ia sair assim que os
servidores se juntassem, dito e feito.
Esses dias eu tava nostálgico com o tempo do BSG e fui caçar vídeo de
alguém tentando fazer exatamente o que você tá fazendo, daí tudo na minha
cabeça se ligou na hora. Te sigo no Twitter, acho que eu sou seu único
seguidor lá

Em sáb, 7 de mar de 2020 02:03, Victor Cesar <notifications@github.com>
escreveu:

**Comment by victti** [comment]
Na vdd n era uma surtur, era uma nidd >.<
Eu nem joguei mt depois daquilo, um tempo depois eu criei uma conta colonial e fiquei por la de vez enquanto. Quando eu fiquei sabendo q o jogo ia fechar, tentei salvar uns arquivo mas a unica coisa de util q salvei foi dados do launcher q iam pro jogo. Eu to fazendo isso daq desde o dia 1 q o jogo fechou >.<
Se vc n acompanha as coisas no Discord, as vezes posto atualizações aq http://battlestar-galactica.online/ ou no meu canal, jaq n da mais pra eu atualizar as coisas no git
Legal saber q vc ainda se interessa no jogo ^^

**Comment by Gr8Jerryto** [comment]
Eu já tinha dado as cartas a um tempinho e pra mim, depois que adicionaram
aqueles 3 novos setores eu mal jogava já, depois recebo no email que jogo
"acabou" tem como me adicionar na steam ou me seguir no twitter pra
facilitar a communicate comunicação? Vou entrar nesse discord

Em seg, 9 de mar de 2020 16:00, Victor Cesar <notifications@github.com>
escreveu:

**Comment by Gr8Jerryto** [comment]
Eu realmemte não esperava te achar denovo, me adiciona, victii, Pep Yeeto
#5773

Em seg, 9 de mar de 2020 16:00, Victor Cesar <notifications@github.com>
escreveu:

**Comment by victti** [comment]
Aq n apareceu nd, tenta vc victti#0401
